configure.py

```diff
@@ -76,7 +76,7 @@ def generate_docker_compose(template_file: str, ip: str, develop: bool = False)
     }
     if develop:
         docker_compose["services"]["dash"]["build"] = "."
-        docker_compose["services"]["dash"]["volumes"] = ["./app:/app"]
+        docker_compose["services"]["dash"]["volumes"] = ["./app:/app", "./logs:/logs"]
```

Comment:
Ok, we're _almost_ there. These logs need to b included in the production version as well as the develop version, which means it needs to be added to where `docker_compose["services"]["dash"]` is defined above this `if develop` block. But only the `logs` volume and not the `app` volume.

I'd suggest adding the something like `"volumes": ["./logs:/logs"]` to the dictionary above and then changing this line to:
```suggestion
        docker_compose["services"]["dash"]["volumes"] += ["./app:/app"]
```